introduction.md

```diff
@@ -1,4 +1,11 @@
 My name is William Hutchison.
 I am a software developer for test.
 I live in Redmond, Washington.
-My favorite CodeSchool path is Electives.
\ No newline at end of file
+My favorite CodeSchool path is Electives.
+Skills:
+* Git
+* GitHub
+* C#
+* Java
+* Testing
+
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?